schemaregistry/serde/protobuf/protobuf.go

```
@@ -148,7 +152,29 @@ func (s *Deserializer) ConfigureDeserializer(client schemaregistry.Client, serde
 	return nil
 }
 
+func (s *Serializer) RegisterDataType(message proto.Message) error {
+	descName := string(message.ProtoReflect().Descriptor().FullName().Name())
+	metadata, ok := s.cachedMetadata[descName]
+	if !ok {
+		messageDesc, err := desc.LoadMessageDescriptorForMessage(protoV1.MessageV1(message))
```

The error is not handled here ^

---

Thanks. Added error check.